files/image_config/constants/constants.yml

```diff
@@ -18,6 +18,18 @@ constants:
       enabled: true
       ipv4: 64
       ipv6: 64
+    allow_list:
+      enabled: true
+      default_action: "permit"   # or "deny"
+      drop_community: 5060:12345  # value of the community to identify a prefix to drop. Make sense only with allow_list_default_action equal to 'permit'
+      default_pl_rules:
+        v4:
+          - "deny 0.0.0.0/0 le 17"
```

Comment:
why cannot i find these default rules in the src/sonic-bgpcfgd/tests/data/general/policies.conf/result_all.conf

Reply:
Because [src/sonic-bgpcfgd/tests/data/general/policies.conf/param_all.conf](https://github.com/Azure/sonic-buildimage/pull/5309/files#diff-afb0b11a15d6d7209273d354099589d5R10) doesn't have default_pl_rules in constants